templates/faqs.html

```diff
@@ -141,8 +141,10 @@
 											<h4><span>Q.</span>I need to cancel my ticket due to unavoidable cicumstances. What is the refund policy?</h4>
 											<div class="tg-panelcontent">
 												<div class="tg-description">
-													<p><span>A.</span>We allow tickets to be cancelled, but cancellation might be barred in the last few days leading up to the conference. This space will be updated if cancellation is barred at any time.
-													</p>
+												<p><span>A.</span>- All tickets are transferable.<br>
+														&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;- Early bird tickets cannot be canceled. They can only be transferred.<br>

```

Comment:
Why use `&nbsp;` and not put the content inside a div and give the div margin/padding?